src/BehaviorsSDKNative/Microsoft.Xaml.Interactions.Design/Properties/Resources.Designer.cs

```diff
@@ -39,7 +39,7 @@ internal class Resources {
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.Xaml.Interactions.Design.Properties.Resources", typeof(Resources).Assembly);
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.Xaml.Interactions.DesignTools.Properties.Resources", typeof(Resources).Assembly);
```

Comment:
>DesignTools [](start = 142, length = 11)

This one should stay as Design

Reply:
I did some work to get all the resx designer files using the correct assembly names. It's now using .Design.